ophyd/epics_compat.py

```diff
@@ -0,0 +1,29 @@
+import time
+import os
+
+__all__ = ['setup',
+           'caput', 'caget',
+           'get_pv',
+           'pv_form',
+           'thread_class']
+
+
+# Vendored from pyepics v3.3.0
+# e33b9290282c93f8dfe0fbe81ced55cbcab99564
+# Copyright  2010  Matthew Newville, The University of Chicago.
+# All rights reserved.
+# Epics Open License
+# see other_licenses folder for full license
+def fmt_time(tstamp=None):
+    "simple formatter for time values"
+    if tstamp is None:
+        tstamp = time.time()
+    tstamp, frac = divmod(tstamp, 1)
+    return "%s.%5.5i" % (time.strftime("%Y-%m-%d %H:%M:%S",
+                                       time.localtime(tstamp)),
+                         round(1.e5*frac))
+
+if os.environ.get('CAPROTO', False):
```

Comment:
Do we want to make the environment variable more descriptive ?  (e.g. `USE_CAPROTO`) or 'namespace' it to ophyd (e.g. `OPHYD_CAPROTO`?  

Just throwing thoughts out!